wayback-core/src/main/java/org/archive/wayback/replay/HttpsRedirectRewriteReplayRendeder.java

```diff
@@ -0,0 +1,169 @@
+package org.archive.wayback.replay;
+
+import java.io.IOException;
+import java.io.OutputStream;
+import java.util.Iterator;
+import java.util.Map;
+import java.util.logging.Logger;
+
+import javax.servlet.ServletException;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+
+import org.archive.wayback.ResultURIConverter;
+import org.archive.wayback.core.CaptureSearchResult;
+import org.archive.wayback.core.CaptureSearchResults;
+import org.archive.wayback.core.Resource;
+import org.archive.wayback.core.WaybackRequest;
+import org.archive.wayback.exception.BadContentException;
+
+/**
+ * rewrite https link to http in http header (redirect)
+ * @author lam
+ */
+public class HttpsRedirectRewriteReplayRendeder extends
```

Comment:
Typo here: HttpsRedirectRewriteReplayRendeder should be HttpsRedirectRewriteReplayRenderer